src/suricata.c

```diff
@@ -2164,6 +2168,10 @@ static int InitRunAs(SCInstance *suri)
         sc_set_caps = TRUE;
     /* Get the suricata group ID to given group ID */
     } else if (suri->do_setgid == TRUE) {
+        if (suri->group_name == NULL) {
+            SCLogError("group name cannot be set to an empty value");
+            return TM_ECODE_FAILED;
+        }
```

Comment:
Same here..

However, I wonder if we should move these checks into `util-privs.c` where some basic validation is already being done, so this is just an extra validation that seems to fit there.

Reply:
move to the file as suggested.